haskell-lsp-types/haskell-lsp-types.cabal

```diff
@@ -1,5 +1,5 @@
 name:                haskell-lsp-types
-version:             0.8.2.0
+version:             0.8.3.0
 synopsis:            Haskell library for the Microsoft Language Server Protocol, data types
```

Comment:
Should this not be bumped to 0.9.0.0 since changing the `Hover` definition is a breaking change?

Reply:
Ok